Changes to Tools section (#41)
ACDguide/BigData: BigData/tools/intro.md

# Available tools and their best use

> There are many languages and packages available to analyse and handle data. Each of them will have its own strengths and weaknesses, this section of the book aims to help identifying which languages/tools are best suited to specific tasks.

Reviewer: ```suggestion
There are many languages and packages available to analyse and handle data. Each has its own strengths and weaknesses, but this section aims to help identify which languages/tools are best suited to specific tasks.

```

Reviewer: Just minor wording suggestions.

To identify the best tool/s for a task the following factors should be considered:

- the data: most languages and packages provide the same or overlapping functionalities, so often the data format, size and the way it is structured determines which tool is best.

> - the kind of analysis: some operations can be performed with several different tools, but for specific tasks you might need a specific tool, also the packages performance might be different for different operations.

Reviewer: ```suggestion
- the kind of analysis: some operations can be performed with several different tools, but for specific tasks you might need a specific tool, and the performance might vary significantly for different operations.

```

> - user experience: the previous experience matters, acquiring new skills is good but can take time, it can also be safer to stick with the same tool as you get to know its limitations better. The community experience is also important, meaning, having others around you that are using the same software and can provide code and help. 

Reviewer: ```suggestion
- user experience: the previous experience matters, acquiring new skills is good but can take time, it can also be safer to stick with the same tool as you get to know its limitations better. The community experience is also important, i.e. it can be useful to have others around you that use the same software and can provide code and help. 

```
